packages/frontend/src/components/NewsDrawer/index.tsx

```diff
@@ -45,26 +45,21 @@ export default function NewsDrawer() {
 
   return (
     <>
-      <Button
-        color="secondary.500"
-        size="xs"
-        variant="link"
-        mr={4}
-        onClick={onOpen}
-      >
-        <Text fontWeight="normal">What's new</Text>
+      <Button colorScheme="secondary" variant="link" mr={6} onClick={onOpen}>
+        <Text>What's new</Text>
+        {localLatestTimestamp !== latestNewsTimestamp && (
+          <Box
+            borderRadius="50%"
+            bg="#C05050"
+            w={1.5}
+            h={1.5}
+            top={2}
+            right={-1}
+            position="absolute"
+          />
+        )}
       </Button>
```

Comment:
is absolute positioning preferred over negative margins?

Reply:
the button shifts with and without the Box if it's not position absolute